doc/howto_release.md

```diff
@@ -106,24 +126,6 @@ Push the commit to the upstream repo:
 git push upstream
 ```
 
-## Create variables
-
-For convenience, create Bash variables with the version update script:
-
-```bash
-# Get VERSION and TAG as variables.
-eval $(./utils/update_version.py status --bash)
-```
-
-Version and tag are the same for all releases:
-
-```bash
-echo "$VERSION"
-echo "$TAG"
-```
-
-If in doubt, run the script without `eval $(...)` to see all the variables created.
-
 ## Create release tag
```

Comment:
I wonder how to see the GHA in line 141 below on command line:

`gh run list --branch releasebranch_8_4`

![image](https://github.com/user-attachments/assets/1a6396e3-f2c9-4f5b-bec9-f4165adb89e2)

This is not identical!

Reply:
I can't test locally with my outdated GH CLI which does not have `--branch` but you can explore the number of items to show (`--limit`). Another thing to explore would be branch versus tag. And then GitHub support. It looks good to me with main and default parameters, though.

Reply:
Well, as said: the CI run simply doesn't show up in the `gh` list. Anyway, I'll opt for the web interface.